ansible/all-in-one/roles/xsnippet/files/nginx.conf

```diff
@@ -0,0 +1,32 @@
+load_module modules/ngx_http_js_module.so;
+
+user  nginx;
+worker_processes  1;
```

Comment:
(continue)

Yeah, I know, it's Nginx we are talking about so memory footprint should be low. It's just.. if you use as a reverse proxy for couple of Rocket workers and to server statics, it doesn't sound right to allocate as many workers as we gave CPU-s. I'd prefer to have some constant based on our estimate usage. :) So thanks for changing this.

Reply:
Sure!